utilities_common/cli.py

```diff
@@ -130,7 +130,8 @@ def __init__(self, db=None):
 
 
         if not self.port_dict:
-            click.echo(message="Warning: failed to retrieve PORT table from ConfigDB!", err=True)
+            if not device_info.is_supervisor():
```

Comment:
Is the function `is_supervisor()` defined. I could not find it in this file [device_info](https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/device_info.py)

Reply:
This function has been defined in the PR which is not merged yet: [multi-asic][cli][chassis-db] Avoid connecting to chassis db when cli commands are executed from linecards #8065

Reply:
> Is the function `is_supervisor()` defined. I could not find it in this file [device_info](https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/device_info.py)

This function has been defined in the PR which is not merged yet: [multi-asic][cli][chassis-db] Avoid connecting to chassis db when cli commands are executed from linecards #8065